src/components/ExploreList.tsx

```diff
@@ -58,58 +57,37 @@ export const ExploreList: React.FC<{
   const currentModel = useCurrentModel()
   const history = useHistory()
   return (
-    <FlexItem pt="xlarge">
-      <List>
+    <FlexItem pt="xlarge" ml="xsmall">
+      <List density={0} color={theme.colors.key}>
         {currentModel &&
+          currentModel.explores &&
           currentModel.explores.map((explore: any) => {
             if (
               !explore.hidden &&
               (!search ||
                 explore.label.toLowerCase().includes(search.toLowerCase()))
             ) {
               return (
-                <ListItem mb="0" key={explore.name}>
-                  <CustomLink
-                    onClick={() => {
-                      history.push(
-                        internalExploreURL({
-                          model: currentModel.name,
-                          explore: explore.name
-                        })
-                      )
-                    }}
-                    className={isActive(
-                      currentExplore,
-                      explore,
-                      loadingExplore
-                    )}
-                  >
-                    {explore.label}
-                  </CustomLink>
+                <ListItem
+                  key={explore.name}
+                  truncate={true}
+                  onClick={() => {
+                    history.push(
+                      internalExploreURL({
+                        model: currentModel.name || '',
+                        explore: explore.name,
+                      })
+                    )
+                  }}
+                  selected={isActive(currentExplore, explore, loadingExplore)}
+                >
+                  {explore.label}
                 </ListItem>
               )
             }
+            return undefined
```

Comment:
Should this be `<></>` or is there a check for `undefined` in a caller?

Reply:
At the end of the day it's the same result. It really depends on the situation. For functions in a component I use undefined. For an early exit from the main render I use `<></>`. I always return JSX from the main render. Can I give a good reason for this? No. I'll do this until I find something that tells me its not a good idea,